### DISCUSSION TITLE
service duration & expire date

### DISCUSSION THREAD

**Discussion by fatehiman** [boxbilling/boxbilling#985] [Q&A] [answered]
hi everyone
how can I define the duration (in month for example) for a hosting product??
for example "1GB, 2Addon domain, 1 year" service should expire automatically 12 months after the activation date.
also before the expiration date, a renewal invoice should be generated automatically.

**Comment by BelleNottelling** [comment, marked as answer]
Hi, I'm not sure if I understand the question. You can configure the product as a recurring fee and it (should) automatically generate an invoice when the time comes. If you want a service that lasts 1 year, and is only renewed once per year, you should only configure a yearly recurring payment.
![image](https://user-images.githubusercontent.com/17304943/129975904-45a5e054-5c3e-4a9a-8712-706626f2194f.png)

**Reply by fatehiman** [reply]
Thanks a lot! You solved my problem.
